src/com/palmergames/bukkit/towny/command/TownyAdminCommand.java

```diff
@@ -1425,6 +1433,89 @@ else if (split[1].equalsIgnoreCase("rank")) {
 						sender.sendMessage(ChatTools.formatCommand(Translation.of("admin_sing"), "/townyadmin nation rank remove [resident] [rank] ", ""));
 						return;
 				}
+			} else if (split[1].equalsIgnoreCase("ally")) {
+				if (split.length < 4)
+					return;

```

Comment:
Should we potentially send a usage message to inform the sender that they are missing args? (Same for the `remove` case as well)